avm/res/network/private-endpoint/README.md

```diff
@@ -653,6 +764,13 @@ A private IP address obtained from the private endpoint's subnet.
 - Required: Yes
 - Type: string
 
+### Parameter: `ipConfigurations.properties.groupId`
+
+The ID of a group obtained from the remote resource that this private endpoint should connect to.
+
+- Required: Yes
```

Comment:
This suggests that the groupId is required, while in the description (in the Bicep template) shows 'optional'. Is that correct?

Reply:
Hey @johnlokerse yeah I saw that and I'm just looking at it why the avm-readme didn't catch that (it was auto-generated and should've fixed it automatically).

Reply:
Keeping it as `Required` for 

- `ipConfigurations.properties.groupId`
- `ipConfigurations.properties.memberName`
- `manualPrivateLinkServiceConnections.properties.groupIds`
- `privateLinkServiceConnections.properties.groupIds`

as discussed and since we want consumers to always pass the groupId.

Reply:
The readme utility updates it, if it was declared in the template itself accordingly. In addition, there 'should' be a test that tests for the keyword to match (i.e., that it's only declared as required if required, etc.).